### DISCUSSION TITLE
Announcement

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#2767] [General]
Ever since May 2020 when I initially released Oqtane 1.0, the open source project has evolved based on the part-time contributions of myself and other community volunteers. The project has grown significantly during the past 3 years and it has now reached a stage where it needs more dedicated focus and resources in order to reach its full potential. So I am excited to announce that I have decided to make a significant career change which will allow me to devote more time to the project and community. 

The current state of Oqtane is very reminiscent of DotNetNuke at an equivalent stage in its lifetime... the open source project is well established, the core functionality is fully developed and mature, and adoption is steadily increasing. That being said, it takes much more than a functional software application to create a successful product ecosystem. Successful products need to provide a wide array of supporting services for their users... services such as documentation, templates, training, support, guidance, etc... They also need to focus on increasing visibility and exposure of the product through marketing and promotional activities. And last, but not least, they need to provide the infrastructure to enable the creation of a viable commercial ecosystem around the product. Developing these services requires dedicated effort, investment, and resources... ingredients which are lacking when relying on a part-time, volunteer approach.

To address the needs outlined above, I will be creating a commercial entity which can serve as the foundation and steward of the Oqtane project as it takes the next step on its journey. Since this is a purely self-funded organic initiative, it will take some time to get this fully up and running. However I am confident the community will see the positive effects of this change very soon, and that the increased focus will accelerate the growth and adoption of the Oqtane project.

**Comment by artmedia** [comment]
this is a very important step, thank you

**Comment by larsontim12** [comment]
Awesome news!  Be careful to not get screwed over for all you hard work!  I have been using your works since the IBuySpy days. (That dates me now!)  Keep up the great work and contributions you make to the development community.

Tim
